spec.go

```diff
@@ -7,10 +21,9 @@ import (
 // Specification returns the connector's specification.
 func Specification() sdk.Specification {
 	return sdk.Specification{
-		Name:        "connectorname",
-		Summary:     "<describe your connector>",
-		Description: "<describe your connector in detail>",
-		Version:     "v0.1.0",
-		Author:      "<your name>",
+		Name:    "weather",
+		Summary: "a weather source connector to get current weather data for a certain city",
+		Version: "v0.1.0",
```

Comment:
We need to pull out the version into a global variable so ldflags can set it (again connected to the changes done in https://github.com/ConduitIO/conduit-connector-template/pull/10).

Reply:
I had to do this cuz we didn't have any releases, so when I built the connector, the version would be something like `-nightly.20230201-1-g929fe12-dirty` and this would fail as a connector because of the version format